test/Feature/AndOrParamsSpec.hs

```diff
@@ -210,12 +221,15 @@ spec actualPgVersion =
     context "used with POST" $
       it "includes related data with filters" $
         request methodPost "/child_entities?select=id,entities(id)&entities.or=(id.eq.2,id.eq.3)&entities.order=id"
-          [("Prefer", "return=representation")]
-          [json|[{"id":4,"name":"entity 4","parent_id":1},
-                 {"id":5,"name":"entity 5","parent_id":2},
-                 {"id":6,"name":"entity 6","parent_id":3}]|] `shouldRespondWith`
-          [json|[{"id": 4, "entities":null}, {"id": 5, "entities": {"id": 2}}, {"id": 6, "entities": {"id": 3}}]|]
-          { matchStatus = 201, matchHeaders = [matchContentTypeJson] }
+            [("Prefer", "return=representation")]
+            [json|[
+              {"id":7,"name":"entity 4","parent_id":1},
```

Comment:
Because I had to add more rows to `data.sql`, some PKs were already used. Some requests had to be adjusted to use other PK ids, that were still available.